app/controllers/api/v1/registration_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::RegistrationController < ApplicationController
+  skip_before_action :authorize!, only: :create
+
+  def create
+    user = User.new(registration_params.merge(provider: "standard"))
+    user.save!
+    render json: user, status: 201
+  rescue ActiveRecord::RecordInvalid
+    render json: user, adapter: :json_api,
+    serializer: ErrorSerializer,
```

Comment:
Still not generating the correct JSON formatting as defined in the spec file.  Need work.